web/modules/custom/do_ai_alt_text/tests/src/Traits/ImageMediaCreationTrait.php

```
@@ -0,0 +1,95 @@
+  /**
+   * Returns the source field name of a media type.
+   *
+   * @param string $media_type_id
+   *   Media type to inspect.
+   *
+   * @return string
+   *   Source field name.
+   */
+  protected function getSourceFieldName(string $media_type_id): string {
+    return \Drupal::entityTypeManager()->getStorage('media_type')->load($media_type_id)->getSource()->getConfiguration()['source_field'];
+  }
```

PHPMD (2.15.0)

[error] The parameter $media_type_id is not named in camelCase. (undefined)

(CamelCaseParameterName)